src/java.base/share/native/libjli/java.c

```diff
@@ -1558,9 +1400,76 @@ ParseArguments(int *pargc, char ***pargv,
 
     *pmode = mode;
 
+    if (!headless) {
+        char *jar_path = NULL;
+        if (mode == LM_JAR) {
+            jar_path = *pwhat;
+        }
+        // not in headless mode. we now set a couple of env variables that
+        // will be used later by ShowSplashScreen()
+        SetupSplashScreenEnvVars(splash_file_path, jar_path);
+    }
+
     return JNI_TRUE;
 }
 
+static void
+SetupSplashScreenEnvVars(const char *splash_file_path, // "-splash:" option value (may be NULL)
+                         char *jar_path // the jar file being launched (may be NULL)
+) {
+
+    // command line specified "-splash:" takes priority over manifest one.
```

Comment:
General comment on comment style. Comments should either be written like sentences and start with a Capital and end with a period; or have neither.

Reply:
I have now updated this PR to follow this suggestion for the new/updated code comments that were introduced in this PR.